tests/modules/nf-core/pretextmap/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PRETEXTMAP } from '../../../../modules/nf-core/pretextmap/main.nf'
+
+workflow test_pretextmap {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true)
+    ]
+
+    PRETEXTMAP ( input )
+}
```

Comment:
Include test with both `bam/sam` and `pairs`

Reply:
I added tests for `bam/cram`. `sam` should not be tested according to James Fellows Yates.